web_disable_browser_autocomplete/static/src/xml/base.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-extend="web.datepicker">
```

Comment:
If i understand correctly, the auto complet will be disabled only for date field ? Is it normal ?

Reply:
@renatonlima resolve this from @legalsylvain please

Reply:
@legalsylvain,

Correcty in this PR disable only for date fields, my original idea was disable for all input fields, but I don't found a way to do this, Do you have any suggestions for that?